tests/simulate/test_hoomd.py

```diff
@@ -0,0 +1,183 @@
+        for pair in pot:
+            pot[pair]['r'] = np.array([1.,2.,3.])
+            pot[pair]['u'] = np.array([2.,4.,6.])
+            pot[pair]['f'] = np.array([-2.,-2.,-2.])
+        return (ens,pot)
+
+    #mock gsd file for testing
+    def create_gsd(self):
+        with gsd.hoomd.open(name=self.directory.file('test.gsd'), mode='wb') as f:
+            s = gsd.hoomd.Snapshot()
+            s.particles.N = 5
+            s.particles.types = ['A','B']
+            s.particles.typeid = [0,0,1,1,1]
+            s.particles.position = np.random.random(size=(5,3))
+            s.configuration.box = [10,10,10,0,0,0]
+            f.append(s)
+        return f
+
+    def test_initialize(self):
+        """Test running initialization simulation operations."""
+        #InitializeFromFile
+        ens,pot = self.ens_pot()
+        f = self.create_gsd()
+        op = relentless.simulate.hoomd.InitializeFromFile(filename=f.file.name)
+        h = relentless.simulate.hoomd.HOOMD(operations=op)
+        sim = h.run(ensemble=ens, potentials=pot, directory=self.directory)
+        self.assertIsInstance(sim[op].neighbor_list, hoomd.md.nlist.tree)
+        self.assertIsInstance(sim[op].pair_potential, hoomd.md.pair.table)
+
+        #InitializeRandomly
+        ens,pot = self.ens_pot()
+        op = relentless.simulate.hoomd.InitializeRandomly(seed=1)
+        h = relentless.simulate.hoomd.HOOMD(operations=op)
+        sim = h.run(ensemble=ens, potentials=pot, directory=self.directory)
+        self.assertIsInstance(sim[op].neighbor_list, hoomd.md.nlist.tree)
+        self.assertIsInstance(sim[op].pair_potential, hoomd.md.pair.table)
+
+    def test_minimization(self):
+        """Test running energy minimization simulation operation."""
+        #MinimizeEnergy
+        ens,pot = self.ens_pot()
+        op = [relentless.simulate.hoomd.InitializeRandomly(seed=1),
+              relentless.simulate.hoomd.MinimizeEnergy(energy_tolerance=1e-7,
+                                                       force_tolerance=1e-7,
+                                                       max_iterations=1000,
+                                                       dt=10)
+             ]
+        h = relentless.simulate.hoomd.HOOMD(operations=op)
+        sim = h.run(ensemble=ens, potentials=pot, directory=self.directory)
+
+    def test_integrators(self):
+        """Test adding and removing integrator operations."""
+        init = relentless.simulate.hoomd.InitializeRandomly(seed=1)
+
+        #BrownianIntegrator
+        ens,pot = self.ens_pot()
+        brn = relentless.simulate.hoomd.AddBrownianIntegrator(dt=0.5,
+                                                              friction=1.0,
+                                                              seed=2)
+        brn_r = relentless.simulate.hoomd.RemoveBrownianIntegrator(add_op=brn)
+        op = [init,brn]
+        h = relentless.simulate.hoomd.HOOMD(operations=op)
```

Comment:
You can just create one HOOMD simulation instance (at the start of this method), and then you can set the operations as a list:
```py
h.operations = [init,brn]
```
Same goes for the other test functions